lib/token-assert.js

```diff
@@ -584,6 +584,14 @@ TokenAssert.prototype.noTrailingSpaces = function(options) {
                 fixed = true;
             }
 
+            if (!fixed && precendingToken && precendingToken.type === 'EOF') {
+                precendingToken = this._file.getPrevToken(precendingToken, { includeWhitespace: true });
+                if (precendingToken && precendingToken.isWhitespace) {
```

Comment:
You have 3 tests in outer if and 2 in inner, are you sure you need all of them?
Can you cover it? That's why coveralls reporting about coverage decrease.